system/Autoloader/Autoloader.php

```diff
@@ -481,4 +486,76 @@ public function loadHelpers(): void
     {
         helper($this->helpers);
     }
+
+    /**
+     * Initializes Kint
+     */
+    public function initializeKint(bool $debug = false): void
+    {
+        if ($debug) {
+            $this->autoloadKint();
+            $this->configureKint();
+        } elseif (class_exists(Kint::class)) {
+            // In case that Kint is already loaded via Composer.
+            Kint::$enabled_mode = false;
+        }
+
+        helper('kint');
```

Comment:
In case that `dd()` or other functions are there in the app code,
define functions that shows nothing. If they are not defined, fatal error occurs.